apps/frontend/src/libs/components/balance-wheel/libs/enums/enums.ts

```diff
@@ -0,0 +1,2 @@
+export { BACKGROUND_COLORS } from "./chart-bg-colors.enum.js";
```

Comment:
```suggestion
export { CHART_BACKGROUND_COLORS } from "./chart-background-colors.enum.js";
```

Reply:
I also use these colors on #216 (but in different order to then use that for slice color based on value). I suggest renaming `CHART_BACKGROUND_COLORS` to `SLICE_COLORS` or `CHART_SLICE_COLORS`. It would be really great

Reply:
I think its also better if the data is also fully type so that we can generate the correct background color for that slice?